### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by andy8810** [Dragon-Crypto-Gaming/DCG-Bug-Reporter#157]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Contact Details

@andy8810

### What happened?

Hi There,

Please see the below suggestions for adjustments and summary of my gameplay experience for Aurum Draconis, testing on the Fuji Network. Happy to provide any additional insight for you. Fight on. Very excited for you and the team for making it this far. Please let me know how I can contribute further in any way.

Gameplay Adjustment Notes/Comments:

1)	Approving transactions for healing bandages is extremely annoying in the first few levels.
2)	I cannot see what % of healing potion I am using at the bottom of the screen to heal when I’m outside of battling in the plays when I hover over the potions. Instead, I have to go into my inventory to do this rather than clicking on the potion on the main screen to figure out how much 15% of healing potion each one is.
3)	Waiting on healing potion transactions took time away from the gameplay, particularly with bandages, which are quite common; however, take a long time to apply 5-6 as they each cause a transaction just to heal up between fights in the Plains.
4)	It would be really nice to see my experience bar at all times, especially when fighting. If it could be up in the top left corner along with my health bar, that would make it easier to understand where I’m at with leveling while fighting.
5)	Would like to see a bigger display with graphics and flashing lights congratulating me on achieving another level. I frequently found myself taking on more damage while fighting in the Plains, only to realize that I had leveled and not used my stat points. Think FFXI, Diablo, Pokemon when you knew right away that you were going to level while mid-fight and when you made it to another level you were so stoked to apply your stat points and get new gear, etc. 
6)	Received a lot of DCAR upfront after battles. Not as much DCAU as I originally expected. Was spending more DCAU to heal up to keep going than I received in Rewards. Ended up getting stuck without enough DCAU to continue forward.
7)	When claiming equipment, you cannot claim potions and armor/weapons in the same batch of items. You have to toggle between the screens and do two separate transactions, which took more time to execute and gas fees.
8)	Healer’s hut didn’t work with enough DCAU. Had to go to the inn and reset.
9)	Any time I needed to do multiple transactions that required gas fees, I found myself getting bored waiting on the transaction to go through. Not say, speed it up, but maybe there is a way to eliminate the number of transactions, as mentioned above. 
10)	I Purchased 2 Bonds at 10 DCAU each. It would be nice to have an APY feature on the DCAR earned in the Claim Reward section.
11)	Not sure if I earned the DCAU back for selling my items in the Marketplace? It would be super helpful to have a window that shows a log of the items you sold, how much, and when they sold. 
12)	It would be nice to see my character’s stature and current gear instead of a blank image. I would love to see the gear I am wearing, similar to Dibalo or WoW. 

My Initial Gameplay Path:

Initially bought 2 DCAU worth of gear from the Marketplace. Just to have a sword and two pieces of armor. I also did 2 bonds (worth 10 DCAU each), which so far has earned me about 8 DCAR. Went straight to the plains and fought my tail off. Any time I received a relatively rare item with strong stat points I would claim those loot materials after all 10 of my battles. Took them over to the Marketplace and listed them for sale at relatively cheap prices, usually under 1 DCAU to get them to sell quickly. Kept fighting and using the Inn to heal up when I ran out of available fights. Noticed that it was costly to use the Healer’s Hut, so I did my best to stay afloat with healing potions until I was ready to recharge at the Inn. I also notice during fights, I was earning mostly DCAR in fights, not as much DCAU as I was in the other test network. After fights, I added new gear that helped fit my attack style build, and increased my stat points as best as possible. I ended up getting stuck at level 4 and ran out of DCAU from straight fighting and healing. Endurance was boosted up 12, yet still struggled in my last couple of fights to survive and used all healing potions. Now I am out of turns and DCAU provided.

Ending Stat Points at Level 4 Before Stalling Out:
Attack: 12
Defense: 9
Speed: 4
Endurance: 12
Luck: 7
Magic: 1
Crafting:1 
Gathering: 1

Spent 0.21 AVAX in roughly 2 hours of gameplay. Started with 25 DCAU and 100 DCAR. I bought 2 Bonds for 10 DCAU each. Have earned 8 DCAR in the last two hours, but cannot swap on the test network yet, I guess. With the 20 DCAU I put into Grom’s Bank, or $158 dollars, and have so far earned 8 DCAR or $0.64 @$0.08 per DCAR. After gameplay, I am down to 0.29 DCAU and cannot heal up at the Inn to move my gaming experience forward past level 4 on my first character I used. So, at the current price of $7.90 per DCAU I spent 5 in about 2 hours (or $40). I also spent 0.21 AVAX at $11.70, which comes out to $2.34 spent in gas fees. In total, I would have spent $42.34 to play this game for 2 hours. With the two bonds I have in place earning some DCAR and I the roughly 3 DCAU pending sales in the market place, a value of $23.70, in total, if everything sells, it has cost me roughly $18.64 to play this game for two hours, and only have earnings of under $1 per hour to get back into the green by tomorrow. Plus I cannot move my character any further along until I spend more DCAU.

Conclusion:

For the “Hack and Slash” character play that I enjoy most in these game types, similar to my Diablo days as a kid, I felt like I ran into a wall without needing to spend more DCAU. There were times when waiting on healing potion transactions took time away from the gameplay, particularly with bandages, which are common, but take a long time to apply 5-6 as they each cause a transaction. I did not experience enough healing potion drops to continue to survive at levels 3-4. Even though I boosted my Luck, I received very few items that truly helped me gain an advantage over the opponents. I ended up using the same weapon I initially purchased in the marketplace the whole time, and I received very little armor drops that truly put me in a better position. I also did not received enough DCAU rewards after fights. More DCAR. This caused me to not be able to keep going to the Inn to recharge. 

I would anticipate that if someone new were to start this game and not make it past level 5, they will ultimately not go back in and play again. It needs to be super easy and receive instant success at the beginning to capture the hearts of the new adventurer. Since I was not able to reset any stat points, my character now sits there, not being strong enough to continue to fight without leveling. Maybe if I didn’t do the bond twice, I would still be able to play; however, I would think that someone who is new to the game, would most likely only try it with a handful of DCAU of about 5. Depending on the price of DCAU at the release of the full game. This also seems scary. If the price explodes, then the cost of DCAU to play would become too expensive for the common player to get in. Hopefully they are tied back to the price so everything doesn’t not cost several dollars to use. At the end of the day, this game rocks. I will continue to support this path you and your team are working on. I cannot wait to see the full release of this, and I hope that I can say that I have contributed to the success of this game. Please do not hesitate to reach out if there is any further information or testing I can provide for you.

### What browsers are you seeing the problem on?

Chrome

### Terms and conditions

- [X] I agree to follow DCG's [terms and conditions](https://dragoncrypto.io/terms-of-use)

**Comment by SivDraconis** [comment]
This is well-thought out and cohesive feedback. I hope the team takes this to heart as I agree with everything you've said, especially:

- DCAU costs are too high - we know that they are going to be shifted to be more realistic, but I would like to see that adjusted ASAP during the Campaign. It's important to keep in mind that the DCAU cost is always going to be pegged to USD, so if DCAU goes up in value, the amount of DCAU required to heal/rest will go down.
- "I received very few items that truly helped me gain an advantage over the opponents." This is a key one for me - the items don't scale up like they do in a game like Diablo. Or perhaps they do, and it just takes a long time to get to higher levels. Either way, it doesn't create stickiness.
- Transactions - I'd prefer the majority of healing potions and bandages just be consumables, not NFTs. I don't think there is a compelling reason to have them as NFTs as their value is quite low.

Again - wonderful feedback.

**Comment by andy8810** [comment]
Thanks for the quick response. I hope this is helpful information. The button for claiming more DCAU is grayed out. Is there a time limit on when you can claim more?

Also, I noticed another issue here while continuing to work my way up to Level 5 with potions only.
When claiming a potion, and when I go to see the gas fee, if the cost is higher, then I cannot reject the transaction. It gets stuck and I cannot go back and change to select either more potions to claim or change up the batch of claimable items. It forces me to stick with my original choice of claiming only one item. We need to find a solution to the higher gas fees. So far, they have mostly been under $0.01; however, recently, in the last hour, I have seen the gas fees for pulling a few potions get as high at $0.07 in gas fees.

**Comment by Polypup** [comment]
Thank you for the feedback. I'll take some time tomorrow to pull some of these out that can get patched right away. 

1. Batched healing. A single transaction to heal with multiple bandages
2. UI changes for 
3. APY in Grohm's Bank
4. New level should be rewarded more graphically. 
5. Rebalancing of rewards and spending (coming soon)
6. It's not possible at this time to batch claim resources, consumables and equipment due to the signatures involved without opening us up for an exploits. We could however allow you to submit the transactions instead of having to wait for one to finish prior to making another transaction.
7. UI update for character stats to show the actual character (probably v1.2)

**Comment by andy8810** [comment]
Maybe there is a way to stack multiple healing potions or bandages together in one transaction? Let's say my health goes down to 691/1,200. If I want to use 5 of my 28 total bandages I have in my inventory to get back up to full health, then maybe there is a way to select 5 bandages on a dropdown menu above the icon, instead of having to sign and spend gas fees for all five transactions separately?

Sorry if some of these come off more as feedback, rather than true bug errors. I am hoping this makes the gameplay experience for first-time users be flawless, fun, rewarding, and worth their time playing when it officially comes out. Happy to help.

Best Regards,

Andrew

**Comment by sam-thewise** [comment]
Thank you heaps for all of the detailed feedback! 

The batched healing is going to do exactly that :) So you can heal to all of your hit points in one transaction or use as many potions/bandages as possible. We've got this in the pipeline for this release.

We look forward to hearing more from you. You should see more of this feedback addressed in the current release too.

Thanks again!

**Comment by andy8810** [comment]
When you click on Grom's Bank, you cannot see your available balance of DCAU and DCAR like you can on the Inventory page. You have to close the window and open the Inventory window in order to figure out how much funds you have available.

**Comment by sezerertugrul** [comment]
During the game, the material names are sometimes incorrectly written on the screen. It fixes itself when I exit the game and re-enter it. ( Chorome )
![DragonEnd1](https://user-images.githubusercontent.com/41703727/212498606-81ea5060-9943-45b7-b6a3-328f6a879c0f.png)
